src/main/java/uk/gov/justice/hmpps/prison/web/config/AuditorAwareImpl.java

```diff
@@ -13,14 +15,15 @@
 @Service(value = "auditorAware")
 public class AuditorAwareImpl implements AuditorAware<String> {
 
-    private final AuthenticationFacade userSecurityUtils;
+    private final AuthenticationFacade authenticationFacade;
 
-    public AuditorAwareImpl(final AuthenticationFacade userSecurityUtils) {
-        this.userSecurityUtils = userSecurityUtils;
+    public AuditorAwareImpl(final AuthenticationFacade authenticationFacade) {
+        this.authenticationFacade = authenticationFacade;
     }
 
+    @NotNull
     @Override
     public Optional<String> getCurrentAuditor() {
-        return Optional.ofNullable(userSecurityUtils.getCurrentUsername());
+        return Optional.ofNullable(StringUtils.substring(authenticationFacade.getCurrentUsername(), 0, 32));
```

Comment:
This file is the only new change. Truncate to fit into the create_user_id column. The 'untruncated' version gets put into one of the audit columns